app/services/injection_response_service.rb

```diff
@@ -0,0 +1,13 @@
+class InjectionResponseService
+  def initialize(json)
+    @response = json.stringify_keys
+    raise ParseError, 'Invalid JSON string' unless @response.keys.eql?(%w[errors uuid messages])
+  end
+
+  def run!
+    claim = Claim::BaseClaim.find_by(uuid: @response['uuid'])
+    return false if claim.nil?
+    ia = InjectionAttempt.create(claim: claim, succeeded: @response['errors'].nil?, error_message: @response['errors'])
+    ia.save
```

Comment:
Assuming this will be executed by a scheduled event do we want to put some error handling with logging on this.   Not sure Sentry will catch errors?!

e.g. 

`rescue ActiveRecord::ActiveRecordError`